example/e2e/processJSONFile.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import path from 'path';
+
+const deepMerge = (target: any, source: any) => {
```

Comment:
I think you can use lodash here to achieve the same thing with something like [_.merge()](https://lodash.com/docs#merge), we have lodash already listed as a dependency